### PR TITLE
Fix to Vaulting Displayed Vaulting Message #3374

### PR DESCRIPTION
**Pull request by halander1**
Didn't do much here. Fixed the sentence structure and changed the reference of Owner in the code to Owner.Name to properly pull the name of the object rather than the object itself.

### CODE REVIEW

**Review by Peptide90 — APPROVED** [review]
Looks good. I assume you've tested it locally? It passed tests anyways I was just curious if it worked as intended.

For future reference too, don't do work on your master repo. Keep your master clean so you can pull the latest upstream master to it and do all your work on branches. That way you can work on more things concurrently and not mess up your repo and delete branches as they get merged.

Good first PR though 👌

File: Content.Server/GameObjects/Components/Movement/ClimbableComponent.cs [hunk 1]
```diff
@@ -185,11 +185,11 @@ private async void TryMoveEntity(IEntity user, IEntity entityToMove)
                 // we may potentially need additional logic since we're forcing a player onto a climbable
                 // there's also the cases where the user might collide with the person they are forcing onto the climbable that i haven't accounted for
 
-                var othersMessage = Loc.GetString("{0:theName} forces {1:theName} onto {2:theName}!", user,
-                    entityToMove, Owner);
+                var othersMessage = Loc.GetString("{0:theName} forces {1:theName} onto the {2:theName}!", user,
+                    entityToMove, Owner.Name);
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Movement/ClimbableComponent.cs:189]: Why is only 1 IEntity using its Name property?

Comment (halander1) [reply Content.Server/GameObjects/Components/Movement/ClimbableComponent.cs:189]: I must have missed a piece. My bad. Do I submit a new commit to fix it?

Comment (metalgearsloth) [reply Content.Server/GameObjects/Components/Movement/ClimbableComponent.cs:189]: Yeah just make a new commit and resolve once it's done. TBH I'm pretty sure I bulldozed it for new physics so it probably won't matter soon.

**Comment by mirrorcult** [comment]
this was caused by new localization, the actual fix is to migrate this Loc.GetString call (and preferably others inside that class) to the new localization system, you can check #3417 for an example and https://hackmd.io/@ss14/localization

**Comment by halander1** [comment]
RIP. When i tested it it seemed my fix seemed to work at least for the self drag and pull.

**Comment by mirrorcult** [comment]
you can just change this pr to do that, don't need to make a new one

**Comment by halander1** [comment]
> you can just change this pr to do that, don't need to make a new one

The big issue is I'm not very skilled at this and need more practice before I can properly understand what is happening and possibly. Although the fix is probably simple I don't think I'll have the time to implement it. I closed the PR so someone else can tackle it more competently.

**Comment by mirrorcult** [comment]
sounds good, if you need help with anything feel free to ask anyone in the discord :)

**Pull request closed without merging**